### PR TITLE
Add support for displaying package version info

### PR DESCRIPTION
**Pull request by hemna**
This patch adds the fetching of the pbr based package version
information to display at exporter startup.

### CODE REVIEW

File: openstack_exporter/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from pbr import version
```
Comment (viennaa) [inline openstack_exporter/__init__.py:1]: I guess this would need to be added to requirements, wouldn't it?

File: openstack_exporter/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+version_info = version.VersionInfo('openstack_exporter')
```
Comment (viennaa) [inline openstack_exporter/__init__.py:3]: Where is the actual Version info maintained?

Comment (hemna) [reply openstack_exporter/__init__.py:3]: The versioning for PBR is in git tags.
https://docs.openstack.org/pbr/latest/user/features.html#version

File: openstack_exporter/__init__.py (hunk 1 above)
Comment (hemna) [reply openstack_exporter/__init__.py:1]: pbr is in the requirements.txt currently.

**Review by viennaa — APPROVED** [review]
Alright, let's go 👍

**Pull request merged**